app/src/main/java/com/mapbox/dash/example/MainActivity.kt

```diff
@@ -668,14 +671,15 @@ class MainActivity : DrawerActivity() {
                         background = if (mode == SidebarMode.Transparent) {
                             Color.TRANSPARENT
                         } else {
-                            ThemeManager.theme.backgroundColor.sidebar.color
+                            Color.BLACK
```

Comment:
Let's use some neutral color with transparency so that it looks fine with both themes:
```suggestion
                            ColorUtils.setAlphaComponent(Color.GRAY, 128)
```